### DISCUSSION TITLE
vmess + tls + ws + cdn is blocking, anyone can help me to solve this problem? thanks

### DISCUSSION THREAD

**Discussion by luogeshibu** [v2fly/v2ray-core#2404] [Q&A]
on 2023-03-08, my v2ray server is blocking , I am in china. I don`t know why my v2ray can not working, can anyone help fix this? Thanks.

**Comment by mydogshitgold** [comment]
Need more information.

Please upload both the server and client V2Ray logs as well as any web server logs, ping results, tcpdump, and the name of your CDN provider.

**Comment by luogeshibu** [comment]
Thanks for you reply, my question is solved. Thank you very much. @mydogshitgold
